app/models/project.rb

```diff
@@ -363,8 +364,11 @@ def build_missing_services
   end
 
   def available_services_names
-    %w(gitlab_ci campfire hipchat pivotaltracker flowdock assembla asana
-       emails_on_push gemnasium slack pushover buildbox bamboo teamcity jira redmine custom_issue_tracker)
+    %w(
+      gitlab_ci composer campfire hipchat pivotaltracker flowdock assembla
+      asana emails_on_push gemnasium slack pushover buildbox bamboo teamcity
+      jira redmine custom_issue_tracker
+      )
```

Comment:
Indent the right bracket the same as the start of the line where the left bracket is.